docs/testing.md

```diff
@@ -2,19 +2,37 @@
 
 ## Quick start:
 
-Many of our browser tests have been rewritten as Python unit tests. The remaining rely on data that is not generated during testing, so please note many of these tests might fail if you do not have a production database locally. If you do, open a new Terminal window or tab and change to the project directory,
+To run browser tests, open a new Terminal window or tab and change to the project directory,
 then tell gulp to start the tests:
 
 ```sh
 gulp build
-gulp test:acceptance --suite=integration
-gulp test:acceptance --suite=content
+gulp test:acceptance
+tox -e acceptance
```

Comment:
Probably should put this in a note, as you don't have to call `gulp test:acceptance` AND `tox -e acceptance`, it's just two ways to run the same thing. Or just drop one of them for simplicity and have one command to rule them all.